Sources/server/SwiftModulesAdapter.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SwiftModulesAdapter.swift
+//  swift-index
+//
+//  Created by Cory Alder on 5/12/16.
+//
+//
+
+import shared
+import HTTPSClient
+import Vapor
+import JSON
+
+struct SwiftModulesAdapter: ThirdPartyIndexAdapter {
+    
+    var name: String = "Swift Modules"
+    
+    func search(_ query: String) throws -> [PackageInfo] {
+        
+        let raw = try self.fetch(query)
+        let results = try self.convert(raw)
+        return results
+    }
+    
+    private func fetch(_ query: String) throws -> JSON {
+        //https://swiftpkgs.ng.bluemix.net/api/search/jay?page=1&items=13
+        
+        let uri = URI(scheme: "https", host:"swiftmodules.com", port:443)
+        let path = "/es/_search?q=\(query)"
```

Comment:
Is there a way to limit the number of results? I only pull up to 5 results from the other indices, so it'd be nice if I could do the same here (we could do the client-side, but we'd be wasting bandwidth pulling something we don't use).